server/src/internalClusterTest/java/org/elasticsearch/cluster/PrevalidateNodeRemovalIT.java

```diff
@@ -196,7 +196,9 @@ public void testNodeRemovalFromRedClusterWithTimeout() throws Exception {
         MockTransportService node2TransportService = (MockTransportService) internalCluster().getInstance(TransportService.class, node2);
         node2TransportService.addRequestHandlingBehavior(
             TransportPrevalidateShardPathAction.ACTION_NAME + "[n]",
-            (handler, request, channel, task) -> { logger.info("drop the check shards request"); }
+            (handler, request, channel, task) -> {
```

Comment:
I'm wondering if we can have it reformat these single-line lambdas as expressions instead of code blocks so it doesn't try to turn a single-line statement into a multi-line one. For example, this should just be:

```java
(handler, request, channel, task) -> logger.info("drop the check shards request")
```

Reply:
The Eclipse formatter doesn't support these kinds of structural changes, it's just a formatter.

Spotless does now appear to support an additional tool called [cleanthat](https://github.com/diffplug/spotless/tree/main/plugin-gradle#cleanthat), which can do automated refactorings (see also the [homepage](https://github.com/solven-eu/cleanthat)). Unfortunately when I tried it, I got some weird dependency conflict error, and decided not to investigate further. I'll leave that to you 😁 I did get IntelliJ to look for all potential fix sites, and there's just over 1000. Also, making the change to remove lambda blocks doesn't always result in more readable code after a format. It needs more investigation 🤷 

I do indeed intend to update `.git-blame-ignore-revs` after merging.